src/Model/Search.php

```diff
@@ -290,4 +290,11 @@ public function getTranslationFilters() {
             function ($v) { return !is_null($v); }
         );
     }
+
+    public static function exactSearchQuery($text) {
```

Comment:
Why make it a static method?

Reply:
This function is also used in two other places that do not build a query for the search engine:
- https://github.com/Tatoeba/tatoeba2/blob/8d702ba27136e45a949a15fd1478ee8e53d53799/src/Model/Table/VocabularyTable.php#L106
- https://github.com/Tatoeba/tatoeba2/blob/8d702ba27136e45a949a15fd1478ee8e53d53799/src/View/Helper/VocabularyHelper.php#L54

To me it is comparable to the static functions in `CurrentUser` but if you prefer having it as a method I will change it.

Reply:
Then it’s okay to keep it as static. I was just being curious. :wink: